docs/data/data-grid/row-grouping/row-grouping.md

```diff
@@ -305,6 +305,22 @@ In the example below:
 If you are dynamically switching the `leafField` or `mainGroupingCriteria`, the sorting and filtering models will not be cleaned up automatically, and the sorting/filtering will not be re-applied.
 :::
 
+## Automatic children selection | Selection propagation
+
+By default, selecting a parent row will not select its children.
+Set the `propagateRowSelection` prop to `true` to achieve the following behavior.
```

Comment:
I thought about the other names like `groupSelectsChildren`, `smartSelection`, etc. none of which seem to convey everything this prop does. `propagateRowSelection` seemed closer since `stopPropagation` is a widely known phenomenon in the context of DOM events, which is also kinda similar in terms of behavior.

Better suggestions are welcome.

Reply:
For me, this name is fine